docs/tutorials/sinks/2_http_sink.md

```diff
@@ -41,8 +42,8 @@ use vector_core::{
 
 # Configuration
 
-First we want to update our config to allow an endpoint to be specified. Add this
-field to the `BasicConfig` struct:
+First we want to update our config to allow an endpoint to be specified. Add
+this field to the `BasicConfig` struct:
 
 ```rust
     /// The endpoint to send HTTP traffic to.
```

Comment:
Borderline troll nit but , should we make this doc comment as descriptive as the ones we have been doing in our auto gen docs effort, specifying that its a full URI and expects an http/https scheme ?

Just to hopefully inspire the readers to have such details in their new sinks.

Reply:
We definitely should!